spec/search-api.json

```diff
@@ -163,6 +213,32 @@
           }
         }
       },
+      "SuggestRequest": {
+        "type": "object",
+        "properties": {
+          "filters": {
+            "$ref": "#/components/schemas/CombinedFilter"
+          },
+          "options": {
+            "type": "object",
+            "properties": {
```

Comment:
It looks like these request options are copied from search request, but start and pageLength are not relevant for suggestions. Instead, a `limit` option might be more relevant..